docs/conf.py

```diff
@@ -325,7 +325,8 @@ def setup(app):
 htmlhelp_basename = 'pyhfdoc'
 
 # sphinx-copybutton configuration
-copybutton_prompt_text = ">>> "
+copybutton_prompt_text = ">>> |\\\\$ |\\.\\.\\. "
```

Comment:
This used to be

```python
copybutton_prompt_text = ">>> |\\\\$ |\\[\\d*\\]: |\\.\\.\\.: |\\.\\.\\. "
```

but as I don't think we're going to see prompts of the form `[12345]: ` or `...: ` I think we can stick with just `>>> `, `$ `, and `... `

Reply:
Ah okay according to the docs https://sphinx-copybutton.readthedocs.io/en/latest/#using-regexp-prompt-identifiers that's for 

> ipython and qtconsole + continuation

but we don't need that for our docstrings.